packages/sdk/CHANGELOG.md

```diff
@@ -22,6 +22,16 @@
 
 - Serialization of nonces with `serializeCIS4RevocationDataHolder` to serialize as little endian.
 
+### Changed
+
+- `getEmbeddedModuleSchema` now uses the module version to determine in which custom wasm sections to look for the schema.
+  It also no longer is `async` because there isn't any need for it to be so.
+- `ConcordiumGRPCClient.getEmbeddedSchema` now delegates to `getEmbeddedModuleSchema` instead of `wasmToSchema`
+  (which was removed as it was just a less capable version of `getEmbeddedModuleSchema`).
+  This means that it returns the complete `RawModuleSchema` instead of only the schema bytes.
+  It also means that it returns `null` instead of an error when no embedded schema was found.
+  
+
```

Comment:
These are breaking changes and should be declared as such

Reply:
done